PC/launcher2.c

```diff
@@ -826,6 +841,72 @@ searchPath(SearchInfo *search, const wchar_t *shebang, int shebangLength)
         return RC_BAD_VIRTUAL_PATH;
     }
 
+    // Make sure we didn't find a reparse point that will open the Microsoft Store
```

Comment:
Let's start by refactoring all this out into another function `ensure_no_redirector_stub` that either returns `0` (okay) or the RC constant that should be returned from here.